_data/mentors.yml

```
@@ -591,3 +591,148 @@
   network:
     - linkedin: https://www.linkedin.com/in/kirthikka-devi-v
     - Medium: https://medium.com/@rithik18
+
+- name: Mona Ahluwalia
+  disabled: false
+  sort: 100
+  hours: 3
+  position:  Talent Director, Chapter 2 
+  type: ad-hoc
+  index: 19
+  location: London, UK
+  bio: |
+    I'm a Recruitment professional and start-up scaling specialist. I can help mentees with the soft-skills development to really make impact in their role and in the workplace.
+  image: assets/images/mentors/mona_ahluwalia.jpg
+  languages: English
+  skills:
+    experience: 5-7 Years
+    years: 7
+    mentee: I can help with all soft-skills related coaching
+    areas:
+      - 
+    languages: 
+    focus:
+      - Switch from IC to management position
+    extra: Resume review, emotional intelligence, communication, interviewing
+  network:
+    - linkedin: https://www.linkedin.com/in/mona-ahluwalia-certrp-7754767a/
+
+- name: Andrew King 
+  disabled: false
+  sort: 100
```

```suggestion
  sort: 10
```


Only ad-hoc active in the month will have sorting 100